Resources/Prototypes/_NF/Catalog/Fills/Items/briefcases.yml

```
@@ -46,7 +46,7 @@
 - type: entity
   parent: SpaceCashExpedition
   id: SpaceCashModerateExpedition
-  name: spesos (20000)
+  name: spesos (20.000)
```

Consistency with en-US cultureinfo.
```suggestion
  name: spesos (20,000)
```